charts/argocd-image-updater/values.yaml

```diff
@@ -21,12 +21,12 @@ fullnameOverride: ""
 namespaceOverride: ""
 
 # -- Create cluster roles for cluster-wide installation.
-## Used when you manage applications in the same cluster where Argo CD Image Updater runs.
-## If you want to use this, please set `.Values.rbac.enabled` true as well.
+# | Used when you manage applications in the same cluster where Argo CD Image Updater runs.
```

Comment:
May I ask what is your idea with those pipe characters after the hashtag?
I think the documentation table will "break" this way.

Reply:
@mkilchhofer good catch 😉